exercism/wordy/wordy.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  WordProblem: class WordProblem {
+    constructor(input){
+      this.input = input,
+      this.operands = {
+        'plus': '+',
+        'minus': '-',
+        'multiplied': '*',
+        'divided': '/'
+      },
+      this.sequence = []
+
+    }
+
+    answer() {
+      let splitString = this.input.replace('?',"")
+      .replace(/by/g,'').split(' ').slice(2)
+      let operandsKeys = Object.keys(this.operands)
+      let match = false
+      operandsKeys.forEach(operand => {
```

Comment:
I *think* the intent of this is to check and see if any of the operands are the input string?  We could write this a little more concisely by filtering for the operands, and checking that the resulting array has a length greater than 0 (here, I'm creating another method whose single responsibility is to make this test, and another method whose single responsibility is to split the input string):
```js
inputHasOperator() {
  const operators = Object.keys( this.operators )

  return inputParts().filter( token => operators.includes( token ) ).length > 0
}

inputParts() {
  return this.input.replace('?',"").replace(/by/g,'').split(' ').slice(2)
}
```
This refactoring helps reduce the complexity of this function by creating smaller functions that have a smaller piece of the overall responsibility of solving this problem.